candidates/urls.py

```diff
@@ -256,6 +256,11 @@
         'name': 'upcoming-elections'
     },
     {
+        'pattern': r'^api/(?P<version>v0.9)/candidates_for_postcode',
+        'view': views.CandidatesAndElectionsForPostcodeView.as_view(),
+        'name': 'candidates-for-postcode'
+    },
```

Comment:
I was wondering if it would be easy to add this view to the `api_router` so that it appears as one of the browseable endpoints at `/api/v0.9`. It looks like that is possible: http://stackoverflow.com/q/30389248/223092 but perhaps it doesn't fit very well with the other (more clearly resource-based) endpoints listed there. Dunno.